### PR TITLE
add configuration for java 21 && RH

### PR DESCRIPTION
**Pull request by RadekCap**
This change allows running testsuite with java 21.

### CODE REVIEW

**Comment by zzambers** [comment]
OK.

Possibly some code for 17 and 21 could be unified later.
Also Testing in GH should be probably updated to 21 after this.

**Comment by RadekCap** [comment]
Yes, it could be merged if remains the same. This is just a start allowing to run test suites.

File: wrapper/configure-vendor-specific-settings.sh [hunk 1]
```diff
@@ -38,6 +42,10 @@ function configureRHSpecificSettings() {
     export JDK_REG_VERSION=17
     export JDK_REG="HKLM\Software\JavaSoft\JDK\\${JDK_REG_VERSION}"
     export JAVA_JDK_JVM_DLL=${JAVA_INSTALL_DIR_REG}\\\\bin\\\\server\\\\jvm.dll
+  elif [[ $(ls "$INPUT_FOLDER" | grep "java-21-openjdk") ]]; then
```
[the remark this answers is not in the document]
Comment (RadekCap) [reply wrapper/configure-vendor-specific-settings.sh:45]: Unfortunately, in this case, it isn't.

This is a general setting BEFORE the Java is installed.

[the remark this answers is not in the document]
Comment (judovana) [reply wrapper/configure-vendor-specific-settings.sh:45]: I thought so,but ` "$INPUT_FOLDER"` name confussed me. ok then. ty!

**Pull request merged**